recipes/gstreamer/config.yml

```diff
@@ -1,13 +1,19 @@
 versions:
-  "1.16.2":
+  "1.20.5":
     folder: all
-  "1.18.0":
+  "1.20.4":
     folder: all
-  "1.18.3":
+  "1.19.2":
+    folder: all
+  "1.19.1":
     folder: all
   "1.18.4":
     folder: all
-  "1.19.1":
+  "1.18.3":
     folder: all
-  "1.19.2":
+  "1.18.0":
+    folder: all
+  "1.16.2":
+    folder: all
+  "1.14.5":
```

Comment:
We usual refrain from adding older version, do you specifically need this one?

Reply:
Yes, since we are working with jetson pack 4.7.x, which has ubuntu 18.04 as base system, which is shipped with 1.14.5.
Evt. we will move to a newer version, but I don't know when.
I can remove it and then keep a local version if preferred.

Reply:
No thats a good reason in my books :)